### PR TITLE
Update the code of conduct to v1.1

### PR DESCRIPTION
**Pull request by sudo-bmitch**
This is based on the contributor-covenant.org v2.1 CoC.

I believe this requires a ToB vote to merge, so I'm holding it in a draft state until that completes.

Vote status:

- [X] Brandon Mitchell
- [x] Vincent Batts
- [ ] Giuseppe Scrivano
- [x] Phil Estes
- [x] Ramkumar Chinchani
- [ ] Sajay Antony
- [x] Samuel Karp
- [x] Derek McGowan
- [ ] Aleksa Sarai

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,64 +1,122 @@
...
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
-## Our Responsibilities
+## Enforcement Responsibilities
 
-Project maintainers are responsible for clarifying the standards of acceptable
-behavior and are expected to take appropriate and fair corrective action in
-response to any instances of unacceptable behavior.
+Community leaders are responsible for clarifying and enforcing our standards of
```
Comment (samuelkarp) [inline CODE_OF_CONDUCT.md:41]: I think we need to define who "community leaders" are.

Comment (sudo-bmitch) [reply CODE_OF_CONDUCT.md:41]: Should this be reverted back to the "Project maintainers" language?

Comment (vbatts) [reply CODE_OF_CONDUCT.md:41]: I think so.

Comment (sudo-bmitch) [reply CODE_OF_CONDUCT.md]: I've updated with the requested change: https://github.com/opencontainers/.github/compare/6c055849008338a74b4ac7904d47548e7281711c..76a12147db64752b48261f23a00330d373f69e9e

**Review by vbatts — APPROVED** [review]

**Review by dmcgowan — APPROVED** [review]

**Review by samuelkarp — APPROVED** [review]

**Pull request merged**